react/src/pages/StartPage.tsx

```diff
@@ -0,0 +1,244 @@
+    | 'startSession'
+    | 'startBatchSession'
+    | 'modelService'
+    | 'startFromExample'
+    | 'startFromURL';
+}
+
+const StartPage: React.FC = () => {
+  const { t } = useTranslation();
+  const { token } = theme.useToken();
+  const webuiNavigate = useWebUINavigate();
+
+  const defaultSummaryElements: {
+    [key in StartItem['id']]: StartItem['data'];
+  } = {
+    createFolder: {
+      content: (
+        <ActionItemContent
+          title={t('start.CreateFolder')}
+          description={t('start.CreateFolderDesc')}
+          buttonText={t('start.button.CreateFolder')}
+          icon={
+            <FolderAddOutlined
+              style={{
+                fontSize: token.fontSizeHeading3,
+                color: token.colorPrimary,
+              }}
+            />
+          }
+          onClick={() => webuiNavigate('/data')}
+        />
+      ),
+    },
+    startSession: {
+      content: (
+        <ActionItemContent
+          title={t('start.StartSession')}
+          description={t('start.StartSessionDesc')}
+          buttonText={t('start.button.StartSession')}
+          icon={
+            <AppstoreAddOutlined
+              style={{
+                fontSize: token.fontSizeHeading3,
+                color: token.colorPrimary,
+              }}
+            />
+          }
+          onClick={() => webuiNavigate('/session/start')}
+        />
+      ),
+    },
+    startBatchSession: {
+      content: (
+        <ActionItemContent
+          title={t('start.StartBatchSession')}
+          description={t('start.StartBatchSessionDesc')}
+          buttonText={t('start.button.StartSession')}
+          icon={
+            <AppstoreAddOutlined
+              style={{
+                fontSize: token.fontSizeHeading3,
+                color: token.colorPrimary,
+              }}
+            />
+          }
+          onClick={() => {
+            const launcherValue: DeepPartial<SessionLauncherFormValue> = {
+              sessionType: 'batch',
+            };
+            const params = new URLSearchParams();
+            params.set('step', '0');
+            params.set('formValues', JSON.stringify(launcherValue));
+            console.log(launcherValue);
+            console.log(params);
+            webuiNavigate(`/session/start?${params.toString()}`, {
+              state: {
+                from: {
+                  pathname: '/start',
+                  label: t('start.StartSession'),
+                },
+              } as SessionLauncherPageLocationState,
+            });
+          }}
+        />
+      ),
+    },
+    modelService: {
+      content: (
+        <ActionItemContent
+          title={t('start.ModelService')}
+          description={t('start.ModelServiceDesc')}
+          buttonText={t('start.button.ModelService')}
+          icon={
+            <AppstoreAddOutlined
+              style={{
+                fontSize: token.fontSizeHeading3,
+                color: token['cyan-6'],
```

Comment:
Let's add a comment to change this with token later. And it would better put the same comment on the similar code below to easily change it.

Reply:
It's provided with the default antd token color, but it would be nice to add a specific token as a provider, I'll leave a comment!